### ISSUE TITLE
ReferenceError: window is not defined using vulcanjs framework

### ISSUE THREAD

**Issue by dchan14** [zksailor534/react-adminlte-dash#5]
is there a fix for this issue i am sure its something to do with ssr any help is greatly appreciated

thanks

**Comment by zksailor534** [comment]
You are correct, it does have to do with Server Side Rendering.  It seems to be a limitation with style-loader that I have yet to find time to investigate or work around.  I asked a question myself regarding this problem here:
[https://github.com/survivejs/ama/issues/15](https://github.com/survivejs/ama/issues/15)
I haven't gone back to read through the chapter he wrote, but maybe that will help.

Supporting SSR is on my to-do list, but I can't say when I'll get to it.  If you, or anyone who reads this, figures out how to make it work, please make a Pull Request or let me know how and I'll make the changes necessary.

**Comment by dchan14** [comment]
thanks for the reply, in vulcan i removed the layout portion from being loaded by the server/main.js file and it works but i have another issue now i know when i used to use adminlte adding a route only refreshed the center panel but now it refreshes the whole page i am not sure if it's a vulcan issue or something i am supposed to do with admin-lte-dash.  would you be able to shed some light?
